Jenkinsfile

```diff
@@ -439,10 +453,9 @@ void TestInVM(deviceMode, deploymentMode, distro, distroVersion, kubernetesVersi
                   --privileged \
                   -e CLUSTER=clear \
                   -e GOVM_YAML=`pwd`/_work/clear/deployment.yaml \
-                  -e TEST_BUILD_PMEM_REGISTRY=${env.REGISTRY_NAME} \
+                  -e TEST_LOCAL_REGISTRY=\$(ip addr show dev docker0 | grep ' inet ' | sed -e 's/.* inet //' -e 's;/.*;;'):5000 \
```

Comment:
@pohly I am not clear at this point. Above you are tagging the images as "localhost:5000/..." and here using docker ip as `TEST_LOCAL_REGISTRY`.  Is it that we have to tag the images also with same ip, something like: `172.17.0.1:5000/.`.

Reply:
`localhost:5000/pmem-csi-driver:canary" and `172.17.0.1:5000/pmem-csi-driver:canary` are the same image. They just get accessed via different IP addresses.

I could have used `172.17.0.1:5000/pmem-csi-driver:canary` also when pushing, but using `localhost` was easier.

Reply:
>  sh "docker run -d -p 5000:5000 --restart=always --name registry registry:2"

So you mean the above docker registry could be accessed by both `localhost` and `172.17.0.1`? Ok.

Reply:
Yes.